.github/workflows/documentation.yml

```
@@ -0,0 +1,38 @@
+name: "Documentation"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
```

The naming scheme is not the same as Doctrine, this needs to be changed.

```suggestion
      - "main"
```

---

I keep that in mind!